caas/jujud-operator-dockerfile

```diff
@@ -20,7 +23,6 @@ RUN pip3 install -r /tmp/wheelhouse/jujud-operator-requirements.txt
 # Finally jujud
 ARG JUJUD_DIR=/var/lib/juju/tools
 WORKDIR $JUJUD_DIR
-COPY jujud $JUJUD_DIR
+COPY jujud /usr/local/bin/
```

Comment:
prefer this is a tmp path like we do for agent conf file in the application operator

Reply:
moved jujud to `/opt/`.